content/blog/sprint-review-2008.md

```diff
@@ -0,0 +1,49 @@
+Title: Wazo Platform 20.08 Released
+Date: 2020-06-08
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2008
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 20.08 release.
+
+## New features in this release
+
+* **Presences**: In Wazo 20.08, the presence status of users takes into account the multiple lines of a user: if one of the lines is talking, the phone status of the user will be talking.
+* **Sounds**: We have increased the limit for uploading sound files to Wazo Platform. The system now accepts sound files and music-on-hold up to 16MB in size. Unfortunately, voicemail greeting messages still have a limitation that forbids files greater that 1M.
```

Comment:
IMO we should remove this section. It was more a configuration than a wazo-platform feature. In fact wazo-platform limit the MOH/sound to 40MB and greetings to <Flask default>. So if someone doesn't use reverse proxy and/or custom reverse, it doesn't care about the 16MB